src/components/atoms/Fieldset/Fieldset.md

```diff
@@ -0,0 +1,20 @@
+### Summary
+
+Use `<Fieldset />` to group checkboxes, radio buttons, or other related form items. `<Fieldset />` is a small wrapper around `<fieldset />` that resets browser defaults.
+It should be used along with `<Legend />` element for accessibility reasons.
```

Comment:
Just a thought, but if it always should be used with Legend, could legend be a mandatory prop ?

Reply:
@thegrinder ☝🏻 what do you think, I think is a good idea 👍🏻 . 
It can in a _minor release_ later though.

Reply:
Normally, you use `legend` within `fieldset` but you don't have to. All these elements do right now is reset the default styling. We can improve it/standardize it in the next releases